app/controllers/median.js

```diff
@@ -0,0 +1,81 @@
+var _       = require("lodash");
+var pg      = require("pg");
+var utils = require('../utils');
+var conn_options = require("../../scripts/redshift-config.json");
+
+var medianController = {
+  process: function(req, res, next){
+    var queryParams = _.pick(req.query, ["state", "race", "sex", "agegroup", "compare"]);
+    utils.validateQueryParams(queryParams, function(err, validateCallback) {
+      if(err) { return next(err); }
+
+      if(compare && compare !== "") {
```

Comment:
And if this is just code that was ported over from lambda, let me know and I'll stop nitpicking. ;)